packages/renderer/src/pages/Account/NodeSettings/NodeLocalForm.tsx

```diff
@@ -67,6 +109,7 @@ export function NodeLocalForm() {
                     defaultMessage="Deamon Address"
```

Comment:
Please change "Deamon" to "Daemon" everywhere in the codebase. I understand the confusion - it is spelled incorrectly in the Figma too :)

Reply:
At least I learned new vocabulary. 😄